scripts/openshiftci-presubmit-integratione2ebenchmark.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# fail if some commands fails
+set -e
+# show commands
+set -x
+
+export CI="openshift"
+export TIMEOUT="30m"
+export GINKGO_VERBOSE_MODE="-v"
+make configure-installer-tests-cluster
+make bin
+mkdir -p $GOPATH/bin
+go get -u github.com/onsi/ginkgo/ginkgo
+export PATH="$PATH:$(pwd):$GOPATH/bin"
+export ARTIFACTS_DIR="/tmp/artifacts"
+export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
+
+# Integration tests
+make test-generic
+make test-cmd-login-logout
+make test-cmd-cmp
+make test-cmd-cmp-sub
+make test-cmd-pref-config
+make test-cmd-watch
+make test-cmd-debug
+make test-cmd-storage
+make test-cmd-app
+make test-cmd-project
+make test-cmd-url
+make test-cmd-push
+make test-cmd-link-unlink
+
+# E2e tests
+make test-e2e-beta
```

Comment:
similar is with integration tests - `make test-e2e-all`

Reply:
done